packages/deployment/package.json

```diff
@@ -26,7 +26,7 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "kes": "~2.0.3",
+    "kes": "~2.1.0",
```

Comment:
Probably we also need to update all package versions, given the updates to @cumulus/deployment - but complete understand if you're waiting for reviews to do that for readability